currency-converter/currency.html

```diff
@@ -47,13 +77,49 @@ <h2>Update Existing Currency Rate</h2>
             <input type="text" id="updateTargetCurrency" required />
             <label for="updateRate">New Rate:</label>
             <input type="number" id="updateRate" step="0.00001" required />
-            <button type="submit">Update Rate</button>
+            <button type="submit" class="button">Update Rate</button>
             <p id="updateConfirmation" class="confirmation"></p>
           </form>
         </section>
       </div>
+      <section>
+        <div class="rate">
+          <h2>Currency Rates</h2>
+        </div>
+        <div id="ratesGrid" class="rates-grid">
+          <table id="ratesTable">
+            <thead>
+              <tr>
+                <th>Base Currency</th>
+                <th>Target Currency</th>
+                <th>Rate</th>
+              </tr>
+            </thead>
+            <tbody>
+              <!-- Rates will be dynamically inserted here -->
+            </tbody>
+          </table>
+        </div>
+      </section>
+      <script src="currency.js"></script>
```

Comment:
Always keep your `script` tags at the end of the `<body>`.

Reply:
script has been moved to the end of the body